acmOpportunities/main.py

```diff
@@ -0,0 +1,379 @@
+import requests
+import os
+import json
+import asyncio
+import discord
+import re
+import psycopg2
+import argparse
+from discord import Webhook
+from typing import List
+from bs4 import BeautifulSoup
+from datetime import date
+from dotenv import load_dotenv
+
+# For using my .ENV files
+load_dotenv()
+
+
+# ----------------- FOR CLI LIBRARY COMMAND -----------------
+
+
+def extract_command_value() -> str:
+    parser = argparse.ArgumentParser(
+        description="Custom command for specifying days for fetching jobs."
+    )
+
+    # Add an argument (the custom command) along with the help functionality to see what the command does
+    parser.add_argument(
+        "--days-needed", type=int, help="The number of days needed to fetch jobs."
+    )
+
+    # Parse the argument and insert into a variable
+    arguments = parser.parse_args()
+
+    # Create a new variable to access the --days-needed command
+    days_needed_variable = arguments.days_needed
+
+    # Return the value from the --days-needed custom command
+    return days_needed_variable
+
+
+# ----------------- FOR POSTGRESS -----------------
+
+
+db_host = os.getenv("DB_HOST")
+db_user = os.getenv("DB_USER")
+db_pass = os.getenv("DB_PASS")
+db_name = os.getenv("DB_NAME")
+db_port = os.getenv("DB_PORT")
+table_name = os.getenv("DB_TABLE")
+
+
+""" Returns the connection with the DB """
+
```

Comment:
I advise switching to [docstrings](https://peps.python.org/pep-0257/) for documenting functions

Reply:
Nice sure. I'll push my changes in a bit

Reply:
Resolved: https://github.com/acmcsufoss/acmOpportunities/pull/1/commits/ab61bc4ad59cbcf21b8df00d42f1b8e9a9bbd854